src/webui/www/private/addtrackers.html

```diff
@@ -4,9 +4,9 @@
 <head>
     <meta charset="UTF-8" />
     <title>QBT_TR(Trackers addition dialog)QBT_TR[CONTEXT=TrackersAdditionDialog]</title>
-    <link rel="stylesheet" href="css/style.css" type="text/css" />
-    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
-    <script src="scripts/lib/mootools-1.2-more.js"></script>
+    <link rel="stylesheet" href="css/style.css?v=${VERSION}" type="text/css" />
+    <script src="scripts/lib/mootools-1.2-core-yc.js?v=${VERSION}"></script>
```

Comment:
It's 3rd party code. Besides, it has its version in its name. Is it intended to be modified without changing its version?

Reply:
You're right, we won't be modifying these- and if we do we'll bump the version. I've added the version number to `clipboard.min.js` and `mocha-yc.js` so that they don't need the version query param.